docs/policy/collab-bearer-tokens.md

```diff
@@ -0,0 +1,168 @@
+Collaborations and Bearer Tokens
+================================
+
+Sites in the OSG grant access to their grid services based on client's association with a specific collaboration
+(i.e. VO) instead of granting access on a per-user basis.
+In the past, this type of access was provided through X.509 proxies with VOMS attributes to demonstrate assocation with
+a collaboration:
+
+-   Users (both human and robots) would request that their collaboration sign their X.509 proxies
+    (usually through a VOMS server),
+-   Or in the case of automated services (i.e. pilot job submission) a collaboration could directly create and sign
+    proxies themselves
+
+Now, sites can authenticate and authorize clients presenting bearer tokens, such as SciTokens or WLCG tokens.
+This document describes how collaborations can issue bearer tokens for the aforementioned use cases in ways that are
+compatible with OSG sites.
+
+Issuers
+-------
+
+To generate bearer tokens, a collaboration must adminster at least one "token issuer" to issue tokens to their users.
+In addition to generating and signing tokens, token issuers provide a public endpoint that can be used to validate an
+issued token,
+e.g. an OSG Compute Entrypoint (CE) will contact the token issuer to authorize a bearer token used for pilot job
+submission.
+
+!!! attention "Token issuer uptime"
+    Due to the centralized nature of bearer token validation, token issuers should be treated as critical, highly
+    available services.
+    Otherwise, a token issuer outage will result in OSG sites being unable to authenticate a collaboration's tokens.
```

Comment:
So every individual pilot and every authenticated file transfer will need to contact the token issuer, and if the issuer is down then neither pilots nor file transfers will work? (Do payloads need to contact the issuer?)

Reply:
> So every individual pilot and every authenticated file transfer will need to contact the token issuer, and if the issuer is down then neither pilots nor file transfers will work?

Correct, this is why ATLAS is holding back on moving all US CEs to 3.6

> (Do payloads need to contact the issuer?)

Nope, only services that authenticate tokens need to contact the issuer

Reply:
You might want to spell that out (just so people know you're not kidding).

Reply:
I'm not sure how I can spell it out further than "tokens can't be authenticated if the issuer is down"?

Reply:
Good point, fixed